docs/plugins/trello-github-integ.md

```diff
@@ -9,33 +9,24 @@ This plugin depends on and can be used together with the `trello` plugin (see do
 `trello-github-integ` plugin can also use `trello` plugin's outputs as input. See the example below:
 
 ```yaml
-tools:
-- name: my-trello-board
-  # name of the plugin
-  plugin: trello
-  dependsOn: ["demo.github-repo-scaffolding-golang"]
-  options:
-    owner: YOUR_GITHUB_USERNAME
-    repo: YOUR_REPO_NAME
-    kanbanBoardName: KANBAN_BOARD_NAME
```

Comment:
Here I think it's better to keep trello plugin config, since trello-github-integ depends on its outputs.